README.md

```diff
@@ -1,2 +1,75 @@
+
+To use this system, you must understand that we cannot make guarantees of any 
+sort with respect to the security and privacy of data that you may store in 
+a solidcommunity.net Solid pod or with respect to the functionality and 
+availability of the system. 
+
+Consequently, Solid Pods on Solidcommunity.net should not be used to store 
+personal data, or any data that is otherwise sensitive, private, confidential, 
+or intended in any way for commercial use.
+
+When you use Solid, information is stored in your Solid pod and in the Solid 
+pods of other users, depending on the actions you take and the apps you use. 
+You and other Solid pod users control who has access to that information. 
+Some information, like your solid profile, is public.
+
+As a user of solidcommunity.net, you have the responsibility to understand what 
+data you are storing in your Pod, whom that data is shared with, and what, 
+if any, data you have made public.
+
+If you add public information to Solid then you must expect this information 
+to be visible to anyone. You must ensure that such data is appropriate, 
+as set out below, and also suitable for the legal jurisdictions and cultural 
+communities in which you are connected. Information shared with a limited set 
+of people must also be written responsibly and read discerningly.
+
+By using solidcommunity.net, you agree not to misuse the system or to assist 
+or enable anyone else to misuse the system. For example, you must not do, 
+or attempt to do, any of the following in connection with solidcommunity.net 
+services:
+
+* violate the law in any way, including by storing, publishing or sharing 
+    illegal material, including material that's fraudulent, defamatory, or misleading;
+* breach or otherwise circumvent any security or authentication measures;
+* interfere with or disrupt any user, host, or network, for example by sending 
+    a virus, overloading, flooding, spamming, or mail-bombing any part of 
+    solidcommunity.net;
+* access, search, or create accounts for solidcommunity.net by any means other 
+    than our publicly supported interfaces (for example, by "scraping" or 
+    creating accounts in bulk);
+* send unsolicited communications, promotions or advertisements, or spam;
+* send altered, deceptive or false source-identifying information, 
+    including "spoofing" or "phishing";
+* promote or advertise products or services other than your own without 
+    appropriate authorization;
+* abuse referrals or promotions to get more storage space than deserved;
+* circumvent storage space limits;
+* sell the services of solidcommunity.net unless specifically authorized to do so;
+* harass or abuse solidcommunity.net system operators, representatives or 
+    agents performing services on behalf of the Solid Project;
+* violate the privacy or infringe the rights of others;
+* Use solidcommunity.net to violate any law or regulation.
+
```

Comment:
I think this generally covers harm to self or others, as well as mis/disinformation, but it might be good to really call it out. See https://github.com/solid/solidproject.org/pull/211#pullrequestreview-437305304 and perhaps see if any specific item from https://github.com/solid/solidproject.org/blob/staging/pages/use-solid/apps.md#apps-inclusion-and-exclusion-criteria can be reused here.

Reply:
@csarven I've created a new issue (#26) for tracking and expanding on this. we received some advice and review on this current policy so i'm going to merge given the urgent need, but the policy points out that we can continue to iterate and adjust it as needed 👍